handbook/benefits.md

```diff
@@ -405,13 +473,206 @@ dbt Labs may, in its sole discretion and unless prohibited by law, require an em
+    - [Skillshare](https://www.skillshare.com/) – Online courses for creators
+    - [Highbrow](http://gohighbrow.com/) – Get 5-minute lessons delivered to your inbox every morning
+    - [Datacamp](https://www.datacamp.com/) – Learn Data Science online
+    - [Treehouse](https://teamtreehouse.com/) – Learn Web Design, Coding, and more
+- Books to learn more about a subject or to dive into how other companies do things; for the People team, this may look like:
+    - [Measure What Matters](https://www.whatmatters.com/the-book/) (OKRs)
+    - [Powerful](https://pattymccord.com/book/) (Netflix Culture)
+    - [Work Rules!](https://www.workrules.net/) (Google Culture)
+- Certifications or professional memberships:
+    - [People Ops Society Membership](https://www.peopleopssociety.com/)
+    - CPA Renewal
+
+### General Guidelines for All Stipends
+
+- All stipend expenses must be processed using your Benepass card or submitted for reimbursement through the Benepass app
+- All transactions over $50 USD or local equivalent require a receipt / documentation
+    - All reimbursements require a receipt / documentation
+- All full time employees are eligible for these stipends in their first month, regardless of their actual hire date.
+- Stipend amounts must be used within their allotted time or the **funds will expire** (ie. Each stipend has as fiscal year (Feb 1-Jan 31) or monthly timeline. Funds not used during that time period will expire and will not roll over)
+    - **Expired Health & Wellness Funds example**: On July 18th your Benepass card is charged for your gym membership for $100, reducing your Health & Wellness available funds to $25. On July 25th you purchase a sleeve of golf balls for $15 on your personal card. On August 1st your Health and Wellness wallet’s remaining $25 from July expire and is then refreshed to $125 for the month of August. August 2nd you submit for a reimbursement for the golf balls and the $15 is reimbursed and deducts the $15 from the available $125 August amount.
+- Stipend amounts will be evaluated on an annual basis (Key factors - benchmark data, purchasing power indices, budget availability)
+- Annual Stipends must be submitted in a timely fashion, best practice is to submit expenses within 30 days.
+    - Past 30 day submissions: while you can still submit these expenses past 30 days, this best practice helps ensure we aren’t back-loading and incurring large sums of reimbursements as we close out a Quarter or Fiscal year and timely submissions help with this!
+- Non-comprehensive [**list of eligible items](https://docs.google.com/spreadsheets/d/1EAmO6874G_FqyVAGJTXG-eU3TNIBuNo8WiEHnwS79vY/edit#gid=0)** (This will consistently be updated from time to time)
+
+### Stipend Amounts by Country
+
+Learn about our stipend amounts by country [here](https://www.notion.so/dbtlabs/Perks-Stipends-60efa8c883aa4354b361baeefb86c2c9?pvs=4#6fe12c7cd08c4886a602a13fd4c3e960).
+
+### How to use Benepass - Perks & Stipend Platform
```

Comment:
Referencing above, we should move this to the beginning of the stipend section, as it's mention throughout. Can we also link to Benepass website?